pkg/serverless/frequency.go

```diff
@@ -0,0 +1,73 @@
+package serverless
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/serverless/flush"
+)
+
+const maxInvocationsStored = 50
```

Comment:
50 seems like a lot of invocations to look back. Worried about it not swapping strategies fast enough when there is a change in traffic

Reply:
Changing it to `10`.